settings.gradle

```diff
@@ -0,0 +1,12 @@
+rootProject.name = "ktlint-gradle-samples"
```

Comment:
You can now write this file using Kotlin if you want to `settings.gradle.kts`

Reply:
Changed in 471d810